src/Microsoft.AspNet.Mvc.Core/ModelBinders/BodyModelBinder.cs

```diff
@@ -57,7 +55,9 @@ public class BodyModelBinder : BindingSourceModelBinder
                 var unsupportedContentType = Resources.FormatUnsupportedContentType(
                     bindingContext.OperationBindingContext.HttpContext.Request.ContentType);
                 bindingContext.ModelState.AddModelError(bindingContext.ModelName, unsupportedContentType);
-                return new ModelBindingResult(null, bindingContext.ModelName, isModelSet: false);
+
+                // Base class will create a ModelBindingResult that exits current ModelBinding loop.
+                return null;
```

Comment:
Why is this changing? 

It seems like it has no tangible effect since the base-class just does what the old code would do.